getTestRule.js

```diff
@@ -1,11 +1,9 @@
 'use strict';
 
 const util = require('util');
+const { basicChecks, lint } = require('stylelint');
 
-function getTestRule(injectedStylelint, options = {}) {
-	const basicChecks = injectedStylelint.basicChecks;
-	const stylelint = injectedStylelint.lint;
-
+function getTestRule(options = {}) {
```

Comment:
We should update the [setup section](https://github.com/stylelint/jest-preset-stylelint#setup) of the docs to reflect this new signature.

Reply:
🤦 Done.